app/services/find_fund_activities.rb

```diff
@@ -0,0 +1,20 @@
+class FindFundActivities
+  include Pundit
+
+  attr_accessor :organisation, :current_user
+
+  def initialize(organisation:, current_user:)
+    @organisation = organisation
+    @current_user = current_user
+  end
+
+  def call
+    funds = policy_scope(Activity.funds, policy_scope_class: FundPolicy::Scope)
+      .includes(:organisation)
+      .order("created_at ASC")
+    unless organisation.service_owner
+      funds = funds.select { |fund| fund.organisation == organisation }
```

Comment:
It would be more efficient to have this as part of the database query rather than the 2 parter of loading the query and then enumerating it. We might be able to keep building the query before loading it early with:

```
    funds = policy_scope(Activity.funds, policy_scope_class: FundPolicy::Scope)
      .includes(:organisation)
      .order("created_at ASC")

    funds = if organisation.service_owner
      funds.all
    else
      funds.where(organisation_id: organisation.id)
    end

    funds
```

It's also a touch more verbose with the `funds.all` when `service_owner` which I think improves the readability a fraction more.

Reply:
I totally got out of the habit of `variable = if ...` syntax bc rubocop is usually against it but it def works here 👍